### ISSUE TITLE
Is it working with AArch64

### ISSUE THREAD

**Issue by pszafer** [Salamek/chromium-kiosk#16]
Hi,
Should your package work well with AArch64 installation of ArchLinuxARM on RPI4?
I installed it and screen was just blinking black, haven't seen anything special in logs, now I'm trying out Arm7 architecture.

**Comment by Salamek** [comment]
@pszafer it is used on RPI3, OrangePi PC+ and x86_64 machines, so it should work... it is blinking when it reaches xorg start?

**Comment by pszafer** [comment]
yes, first there was issue with missing `systemd-homed is not available`, when I fixed it, it was only screen blinking.
in processes I see that startx process is working and child process of it are restarting every second (eg chromium)

**Comment by Salamek** [comment]
@pszafer ok, just doing upgrade on my rpi3 with ArchlinuxARM, i'll fix it ASAP

**Comment by Salamek** [comment]
@pszafer well i fixed xorg restart issue in v 0.4.2, but my chromium crashes with `[380:396:0509/223958.523669:FATAL:gpu_data_manager_impl_private.cc(415)] GPU process isn't usable. Goodbye.` under archlinuxArm... (maybe some mess in deps, my install is pretty old etc) it may work ok for you

**Comment by Salamek** [comment]
@pszafer good to hear, that missing dependency will be fixed in next release (0.4.3 if CI wont fuck up)
